src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,12 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
         // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(4);
```

Comment:
Consider using a try-with-resources or a try-finally block to ensure that the ExecutorService is properly shut down even if an exception occurs. This helps to prevent potential resource leaks.